### PR TITLE
Add slice machine 4

### PR DESCRIPTION
**Pull request by gestchild**
Relates to #10664 

- Renders the body slices with SliceZone behind a toggle
- Renders the Standfirst with the untransformed data behind a toggle
- adds the slice-simulator page and some more slice screenshots
- adds some TODO comments for when we've completed the switch to slice machine
- Fixes errors that showed up in storybook

### CODE REVIEW

File: content/webapp/components/Body/Body.tsx [hunk 1]
```diff
@@ -166,12 +171,23 @@ const Body: FunctionComponent<Props> = ({
...
 
+  const filteredUntransformedBody = untransformedBody
+    .filter(
+      slice =>
+        !(
+          slice.slice_type === 'editorialImage' &&
+          slice.slice_label === 'featured'
+        )
+    )
+    .filter(slice => slice.slice_type !== 'standfirst');
```
Comment (rcantin-w) [inline content/webapp/components/Body/Body.tsx:189]: Why are those two filtered?

File: content/webapp/components/SeasonsHeader/index.tsx [hunk 2]
```diff
@@ -16,6 +17,7 @@ import PageHeaderStandfirst from '@weco/common/views/components/PageHeaderStandf
 import PrismicImage from '@weco/common/views/components/PrismicImage/PrismicImage';
 import Space from '@weco/common/views/components/styled/Space';
 import { WobblyBottom } from '@weco/common/views/components/WobblyEdge';
+import Standfirst from '@weco/common/views/slices/Standfirst/index';
```
Comment (rcantin-w) [inline content/webapp/components/SeasonsHeader/index.tsx:20]: np
```suggestion
import Standfirst from '@weco/common/views/slices/Standfirst';
```

File: content/webapp/pages/articles/[articleId].tsx [hunk 3]
```diff
@@ -41,6 +41,8 @@ import { Pageview } from '@weco/common/services/conversion/track';
 import { createPrismicLink } from '@weco/common/views/components/ApiToolbar';
 import { setCacheControl } from '@weco/content/utils/setCacheControl';
 import { isNotUndefined } from '@weco/common/utils/type-guards';
+import { useToggles } from '@weco/common/server-data/Context';
+import Standfirst from '@weco/common/views/slices/Standfirst/index';
```
Comment (rcantin-w) [inline content/webapp/pages/articles/[articleId].tsx:45]: or does it get automatically imported like that? 🤔

File: content/webapp/services/prismic/transformers/body.ts [hunk 4]
```diff
@@ -40,7 +42,7 @@ import {
   ContentListProps,
   Slice,
   Weight,
-} from '@weco/content/types/body';
+} from '@weco/content/types/body'; // TODO should we rename move these transformed types
```
Comment (rcantin-w) [inline content/webapp/services/prismic/transformers/body.ts:45]: rename to what?

File: content/webapp/components/Body/Body.tsx (hunk 1 above)
Comment (gestchild) [reply content/webapp/components/Body/Body.tsx:189]: This is a good question and something I'm trying to understand myself at the moment. There is quite a bit of logic involved around pulling things out of the body and rendering them elsewhere under certain circumstances and I think there's room to improve a lot of this. But first step is understanding exactly what is happening and why.

The standfirst is straightforward in that we now render it separately in the header, so don't want it in the body. The featured stuff looks a bit more complicated. David created a [ticket about potential remodelling](https://github.com/wellcomecollection/wellcomecollection.org/issues/10753), I'll add notes to it when I'm confident what needs doing.

File: common/views/slices/EditorialImage/index.tsx [hunk 5]
```diff
@@ -23,6 +23,8 @@ const EditorialImageSlice: FunctionComponent<EditorialImageSliceProps> = ({
   const transformedSlice = transformEditorialImageSlice(slice);
 
   const options = { ...defaultContext, context };
+  // TODO: use one layout for all image weights if/when it's established
+  // that width isn't an adequate means to illustrate a difference
```
Comment (rcantin-w) [inline common/views/slices/EditorialImage/index.tsx:27]: Do we have a ticket for that?

**Review by rcantin-w — APPROVED** [review]
Nice one! 🙌

File: common/views/slices/EditorialImage/index.tsx (hunk 5 above)
Comment (gestchild) [reply common/views/slices/EditorialImage/index.tsx:27]: not yet

**Pull request merged**